e2e/tests-dfx/install.bash

```diff
@@ -81,3 +81,19 @@ teardown() {
     dfx canister create --all
     assert_command_fail dfx canister install --all --wasm "${archive:?}/wallet/0.10.0/wallet.wasm"
 }
+
+@test "install runs post-install tasks" {
+    dfx_start
+    # shellcheck disable=SC2094
+    cat <<<"$(jq '.canisters.e2e_project."post-install"="sh -c \"echo hello\""' dfx.json)" >dfx.json
```

Comment:
It looks like SerdeVec is meant to support the post-install field being either an array or a string.  Can we have an e2e test that demonstrates expected behavior when the `post-install` field is an array?